policies/coding-style.md

```diff
@@ -200,6 +200,9 @@ such as this partial list:
     =  +  -  <  >  *  /  %  |  &  ^  <=  >=  ==  !=  ?  : +=
 ```
 
+Put a space after commas
+and after semicolons in `for` statements, but not in `for(;;)`.
```

Comment:
`` `for(;;)` `` → `` `for (;;)` ``

(see line 164)

Reply:
Oops, I was giving a bad model - should have used `check-format.pl` ;-)